app/views/admin/order_cycles/checkout_options.html.haml

```diff
@@ -78,11 +78,11 @@
                   %p
                     = t('.no_payment_methods')
 
-  %div{"data-checkbox-selection-visibility":""}
-    This is a warning alert—check it out!
 
   %div#save-bar
     %div.container
+      %span.error-message{"data-checkbox-selection-target":"errorMessage"}
+        Please select at least one shipping method and one payment method.
```

Comment:
Any text should be translated with I18n.

Reply:
I think this would be the section to add the message: https://github.com/openfoodfoundation/openfoodnetwork/blob/df8283661d841c83e17f73d3be7b1fd88dc612f2/config/locales/en.yml#L1254-L1265
These messages can be included in the template with the `t()` helper method (as is done elsewhere in this file).